README.md

```diff
@@ -436,8 +438,42 @@ Available form helpers:
 * password_field
 * file_field
 * select
+* multiselect ([Stimulus-based](https://github.com/Purple-Magic/tramway#stimulus-based-inputs))
 * submit
 
+#### Stimulus-based inputs
+
+`tramway_form_for` provides Tailwind-styled Stimulus-based custom inputs.
+
+##### Multiselect
+
+In case you want to use tailwind-styled multiselect this way
+
+```haml
+= tramway_form_for @user do |f|
+  = f.multiselect :permissions, [['Create User', 'create_user'], ['Update user', 'update_user']]
+  #- ...
+```
+
+you should add Tramway Multiselect Stimulus controller to your application.
+
+Example for [importmap-rails](https://github.com/rails/importmap-rails) config
+
+*config/importmap.rb*
+```ruby
+pin '@tramway/multiselect', to: 'tramway/multiselect_controller.js'
+```
+
+*app/javascript/controllers/index.js*
+```js
+import { application } from "controllers/application"
+import { eagerLoadControllersFrom } from "@hotwired/stimulus-loading"
+import { Multiselect } from "@tramway/multiselect" // importing Multiselect controller class
+eagerLoadControllersFrom("controllers", application)
+
+application.register('multiselect', Multiselect) // register Multiselect controller class as `multiselect` stimulus controller
```

Comment:
Is it possible to change the controller name if an application already has "multiselect" controller?

Reply:
Nice catch! It's not possible right now.

I've created [a new task](https://github.com/orgs/Purple-Magic/projects/2/views/1?pane=issue&itemId=79167803) for this case.